src/components/Header/Header.jsx

```diff
@@ -159,29 +112,30 @@ const Header = ({
       );
     }
     return (
-      <StyledGlobalAction
+      <HeaderGlobalAction
+        className="header-action-btn"
```

Comment:
Do we need to begin to bake in the `$prefix` to this classname?

Reply:
i had thought about this but was worried that it may someday be used by carbon and over ride our styles.

Reply:
Not opposed to adding it though